src/test/java/it/okkam/validation/FiscalCodeValidatorTest.java

```diff
@@ -54,8 +55,10 @@ public void testComuniMapInitialization() throws IOException {
   @Before
   public void init() throws IOException {
     String codiciIstatStr = readLocalFile(CODICE_ISTAT_COMUNI_CSV);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Write to static field it.okkam.validation.FiscalCodeValidatorTest.conf1 from instance method it.okkam.validation.FiscalCodeValidatorTest.init() [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=findbugs%3AST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD)